src/measure_theory/function/lp_space.lean

```diff
@@ -1213,18 +1213,36 @@ end
 
 end has_measurable_add
 
-section normed_space
+/-!
+### Bounded actions by normed rings
 
-variables {𝕜 : Type*} [normed_field 𝕜] [normed_space 𝕜 E] [normed_space 𝕜 F]
+In this section we show inequalities on the norm.
+-/
+section has_bounded_smul
 
-lemma snorm'_const_smul {f : α → F} (c : 𝕜) (hq_pos : 0 < q) :
-  snorm' (c • f) q μ = (‖c‖₊ : ℝ≥0∞) * snorm' f q μ :=
+variables {𝕜 : Type*} [normed_ring 𝕜] [mul_action_with_zero 𝕜 E] [mul_action_with_zero 𝕜 F]
+variables [has_bounded_smul 𝕜 E] [has_bounded_smul 𝕜 F]
+
+namespace lipschitz_with
```

Comment:
all this section about `lipschitz_with` has nothing to do with `has_bounded_smul`, right? Could you move it above it?
Also, `lipschitz_with`-like results appear later in this file (see `lipschitz_with.comp_mem_ℒp` for instance). Are your lemmas useful to streamline things there, or is completely orthogonal?

Reply:
I think you're right; unfortunately there's a bit of a dependency web to untangle here.

Reply:
#19092 untangles these results a bit. I don't think `lipschitz_with.comp_mem_ℒp` helps all that much, but I'll have one last go at unifying it after #19092 is in.

Reply:
I just removed these lemmas as I didn't actually need them any more.